Content.Server/GameTicking/GameTicker.cs

```diff
@@ -635,11 +645,22 @@ public void EquipStartingGear(IEntity entity, StartingGearPrototype startingGear
             {
                 foreach (var slot in AllSlots)
                 {
-                    var equipmentStr = startingGear.GetGear(slot, profile);
+                    var equipmentStr = startingGear.GetGear(slot, profile, CaptainUniformPreference);
                     if (equipmentStr != "")
                     {
                         var equipmentEntity = _entityManager.SpawnEntity(equipmentStr, entity.Transform.Coordinates);
-                        inventory.Equip(slot, equipmentEntity.GetComponent<ItemComponent>());
+                        ItemComponent itemComponent;
+                        try
+                        {
```

Comment:
I added this check here, because even without changes I was getting an error that was crashing the server. Figured if an entity wasnt correct it should just not be spawned instead of kil

Reply:
"NEVER use try catch for getting components" update this

Reply:
i did it like chat asked

Reply:
In this instance probably better to deal with the exceptions if they were popping up from your pr rather than tryget ngl.